configuration/configuration.go

```diff
@@ -0,0 +1,96 @@
+package configuration
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/spf13/viper"
+)
+
+const (
+	// DefaultConfigFilePath is the path to the configuration file that is used if no other
+	// file is specified via the -config switch or via the ALMIGHTY_CONFIG_FILE_PATH environment
+	// variable.
+	DefaultConfigFilePath = "config.yaml"
+)
+
+// GetConfiguration returns the current configuration as a string
+func GetConfiguration() string {
```

Comment:
I understand. But since the real configuration is done with viper and we don't have any object to attach this method to it would result in a call `configuration.AsString()` is this what you want? Also, I think in Go it should be `configuration.String()`.